cmd/neofs-node/config.go

```diff
@@ -954,7 +968,7 @@ func (c *cfg) configWatcher(ctx context.Context) {
 			}
 
 			for _, component := range components {
-				err = component.cfg.Reload()
+				err = component.reload()
```

Comment:
is it a good idea to call it in such a way? looks like it is a `component`'s `reload` method. do not really think it is bad but anyway
mb call it `reloadFunc`?

on the other hand: why isn't a `Reload` method like for the `logger`?

Reply:
Renamed reload -> reloadFunc